### PR TITLE
Add an install generator

### PR DESCRIPTION
**Pull request by blocknotes**
## Description
Add an install generator.

## Install test procedure

- Build our gem: `gem build solidus_starter_frontend.gemspec`
- Install it in the system: `gem install solidus_starter_frontend-0.0.1.gem`
- Create a new Rails app using scaffolding:
`rails new solidus_test1 --skip-action-cable --skip-bootsnap --skip-coffee --skip-spring --skip-system-test --skip-test --skip-turbolinks`
- Add to the Gemfile:
```ruby
gem 'solidus_core'
gem 'solidus_api'
gem 'solidus_backend'
gem 'solidus_sample'
```
- Execute `bundle`
- Execute `bin/rails g spree:install`
- Execute `solidus_starter_frontend`

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,32 +28,26 @@ exclude it and manually include all the other Solidus componenents.
...
+Execute our generator that will copy our component files in your project:
+`solidus_starter_frontend`
 
-If it's the first Solidus installation you need to run the Solidus install
-before `bundle exec rails g solidus_starter_frontend:install`.
-Please, take a look at the [Solidus README](https://github.com/solidusio/solidus#installation-options).
+If Solidus was already installed with solidus_frontend you will have to change
+all `Spree::Frontend::Config` in `SolidusStarterFrontend::Config`.
 
 ## Customization
```
Comment (ChristianRimondi) [inline README.md:52]: Is this section still required? 🤔

Comment (blocknotes) [reply README.md:52]: That section is referring to switching from `solidus_frontend` to our component.
I would keep it if we still consider this option feasable.

Comment (blocknotes) [reply README.md:52]: Ahh... you mean the Customization section.
Sure, I'm going to remove it 👍

Comment (blocknotes) [reply README.md]: New commit added

**Review by kennyadsl — APPROVED** [review]
👍 This works for me.

**Pull request merged**